xpk.py

```diff
@@ -1180,12 +1224,11 @@ def run_gke_node_pool_create_command(args, system_characteristics) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
-
+  device_type, acc_type = get_device_and_accelerator_type(args)
```

Comment:
You can remove this line and get the acceleratortype from system_characteristics.